site/content/docs/5.2/getting-started/parcel.md

```diff
@@ -1,101 +1,149 @@
 ---
 layout: docs
-title: Parcel
-description: Learn how to include Bootstrap in your project using Parcel.
+title: "Bootstrap & Parcel"
+description: The official guide for how to include and bundle Bootstrap's CSS and JavaScript in your project using Parcel.
 group: getting-started
 toc: true
 ---
 
-## Install Parcel
+{{< callout >}}
+**Want to skip to the end?** Download the source code and working demo for this guide from the [twbs/examples repository](https://github.com/twbs/examples/tree/main/parcel). You can also [open the example in StackBlitz](https://stackblitz.com/github/twbs/examples/tree/main/parcel?file=index.html), but Parcel isn't currently supported there.
```

Comment:
```suggestion
**Want to skip to the end?** Download the source code and working demo for this guide from the [twbs/examples repository](https://github.com/twbs/examples/tree/main/parcel). You can also [open the example in StackBlitz](https://stackblitz.com/github/twbs/examples/tree/main/parcel?file=index.html) but not run it because Parcel isn't currently supported there.
```

Reply:
Suggested a change (something like that) because at first it wasn't obvious for me to understand why there was a link to StackBlitz and saying at the same time that Parcel wasn't supported.